packages/client/src/views/CorrectionForm/CorrectionSummary.tsx

```diff
@@ -258,25 +258,6 @@ class CorrectionSummaryComponent extends React.Component<IFullProps, IState> {
               noResultText={intl.formatMessage(constantsMessages.noResults)}
             ></Table>
 
-            <Table
-              hideTableBottomBorder={true}
-              isLoading={false}
-              content={[
-                {
-                  idCheck: this.getIdCheck()
-                }
-              ]}
-              columns={[
-                {
-                  label: intl.formatMessage(messages.correctionSummaryIdCheck),
-                  width: 100,
-                  alignment: ColumnContentAlignment.LEFT,
-                  key: 'idCheck'
-                }
-              ]}
-              noResultText={intl.formatMessage(constantsMessages.noResults)}
-            ></Table>
-
```

Comment:
@favour234 We want to conditionally render this `Table` and not hide it entirely. When the correction is requested by `Another registration agent or field agent` or `Me`, only then should we hide it

Reply:
alright..would update..

Reply:
updated